HW1/P2/P2b.py

```diff
@@ -1,3 +1,25 @@
+import pyspark
+from pyspark import SparkContext
+sc = SparkContext()
 from P2 import *
+import random 
 
-# Your code here
+# make two ranges of 2000 numbers and random shuffle on both
+x = range(2000)
+y = range(2000)
+random.shuffle(x)
+random.shuffle(y)
+
```

Comment:
In this way, although the shuffle time increases a little bit, the time of calculation decreases a lot, which makes the total time decrease. Good job!